nengo_spinnaker/netlist/netlist.py

```diff
@@ -146,7 +148,7 @@ def place_and_route(self, system_info,
                             constraints, extended_placements,
                             extended_allocations, **route_kwargs)
 
-    def load_application(self, controller):
+    def load_application(self, system_info, controller):
```

Comment:
Since this commit otherwise looks entirely sensible I'll waste your time by asking why system info before MC in the arguments? ;P